README.md

```diff
@@ -490,3 +492,5 @@ make build-doc
 
 This generates the Doxygen documentation under `build-doc/doxygen` folder.
 Run for example: `firefox build-doc/doxygen/index.html &` to view them.
+
+NOTE! If you have a `snap`-based firefox in use, it will not access rights to show ocal files anymore.
```

Comment:
```suggestion
NOTE! If you have a `snap`-based firefox in use, it will not have access rights to show local files anymore.
```

Reply:
Has this been applied? It's marked as resolved but still looks like the original.